agent/src/main/java/com/microsoft/applicationinsights/agent/internal/agent/http/HttpClassDataProvider.java

```diff
@@ -155,7 +164,9 @@ public MethodVisitor create(MethodInstrumentationDecision decision,
         	throw td;
         } catch (Throwable t) {
             try {
-                InternalAgentLogger.INSTANCE.error("Exception while loading HTTP classes: '%s'", t.toString());            } catch (ThreadDeath td) {
+                InternalLogger.INSTANCE.error("Exception while loading HTTP classes: '%s'",
+                    ExceptionUtils.getStackTrace(t));            }
```

Comment:
Spacing

Reply:
Probably for this IDE did a better job to not have a line with 150 + characters :) anyway again a style guide issue.